src/lib/stores/verify-module.ts

```diff
@@ -31,37 +34,61 @@ export class VerifyModuleTaskStore {
     return isHydrated(this);
   }
 
-  isVerifyModuleTaskUserKeyExist(): boolean {
+  isMoveVerifyTaskUserKeyExist(): boolean {
     return !!this.userKey;
   }
 
-  setVerifyModuleTaskUserKey(userKey: string) {
+  setMoveVerifyTaskUserKey(userKey: string) {
     this.userKey = userKey;
   }
 
-  isVerifyModuleTaskExist(taskId: string): boolean {
+  isMoveVerifyTaskExist(taskId: string): boolean {
     return (
-      this.getVerifyModuleTasks().findIndex((item) => item.taskId === taskId) >
-      -1
+      this.getMoveVerifyTasks().findIndex((item) => item.taskId === taskId) > -1
     );
   }
 
-  getVerifyModuleTasks(): VerifyModuleLocalInfo[] {
-    return this.modules[this.userKey]?.reverse() ?? [];
+  getMoveVerifyTasks(): MoveVerifyTaskLocalInfo[] {
+    return this.modules[this.userKey] ?? [];
   }
 
-  getVerifyModuleTask(taskId: string): VerifyModuleLocalInfo | undefined {
-    return this.getVerifyModuleTasks().find(
-      (module) => module.taskId === taskId
-    );
+  latestMoveVerifyTasks(): MoveVerifyTaskLocalInfo[] {
+    return this.getMoveVerifyTasks()?.slice().reverse();
```

Comment:
remove ?

Reply:
It was used in `useMyModuleVerifications`.